config/install/core.entity_view_display.taxonomy_term.sites.default.yml

```diff
@@ -146,6 +158,14 @@ content:
     third_party_settings: {  }
     region: content
     type: entity_reference_label
+  field_title_of_table_of_contents:
+    weight: 15
+    label: above
+    settings:
+      link_to_entity: false
+    third_party_settings: {  }
+    type: string
+    region: content
 hidden:
```

Comment:
@joshua-salsadigital I don't see the changes from this file is getting updated using the update hook here. Please correct me if I am wrong